### PR TITLE
chore(clerk-react): Update docs

### PR DESCRIPTION
**Pull request by devchampian**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [x] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/types`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Added the following documentation to this package repo:

- Updated README
- Updated LICENSE
- docs/CODE_OF_CONDUCT.md
- docs/CONTRIBUTING.md
- docs/SECURITY.md

### CODE REVIEW

File: packages/react/README.md [hunk 1]
```diff
@@ -1,7 +1,128 @@
...
+  ClerkProvider,
+  SignedIn,
+  SignedOut,
+  SignInButton,
+  UserButton,
+} from "@clerk/clerk-react";
+
+const frontendApi = process.env.REACT_APP_CLERK_FRONTEND_API;
+
+render(
```
Comment (igneel64) [inline packages/react/README.md:76]: ❓ 
This example seems really unfamiliar to me. The only place I see `render` called directly is in the `index` file of CRA type apps. Can we change it and just add the provider on the `App` component ?
WDYT?

File: packages/react/README.md [hunk 2]
```diff
@@ -1,7 +1,128 @@
...
+
+You can get in touch with us in any of the following ways:
+
+- Join our official community [Discord server](https://discord.com/invite/b5rXHjAg7A)
+- Open a [GitHub support issue](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=question&template=ask_a_question.md&title=Support%3A+)
+- Contact options listed on [our Support page](https://clerk.dev/support?utm_source=github&utm_medium=clerk_react)
+
+## Contributing
+
+We're open to all community contributions! If you'd like to contribute in any way, please read [our contribution guidelines](docs/CONTRIBUTING.md).
```
Comment (igneel64) [inline packages/react/README.md:110]: 🙃 
For links that might be usable on the npm page context, can we use the full URLs pointing to the main branch ? 
This way they are accessible from NPM also. WDYT ?

File: packages/react/README.md [hunk 3]
```diff
@@ -1,7 +1,128 @@
...
+
+`@clerk/clerk-react` is provided **"as is"** without any **warranty**. Use at your own risk.
+
+_For more information and to report security issues, please refer to our [security documentation](docs/SECURITY.md)._
+
+## License
+
+This project is licensed under the **MIT license**.
+
+See [LICENSE](LICENSE) for more information.
```
Comment (igneel64) [inline packages/react/README.md:124]: 🙃 
Same as above _(link is missing here)_

File: packages/react/docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+1. Search our repository for open or closed
+   [Pull Requests](https://github.com/clerkinc/javascript/pulls)
+   that relate to your submission. You don't want to duplicate effort.
+2. Fork the project
+3. Create your feature branch (`git checkout -b feat/amazing_feature`)
+4. Commit your changes (`git commit -m 'feat: Add amazing_feature'`). Clerk uses [conventional commits](https://www.conventionalcommits.org), so please follow the specification in your commit messages.
+5. Push to the branch (`git push origin feat/amazing_feature`)
+6. [Open a Pull Request](https://github.com/clerkinc/javascript/compare?expand=1)
+
+## Publishing packages
```
Comment (igneel64) [inline packages/react/docs/CONTRIBUTING.md:73]: 🙃 
IMO I would skip the next two sections and keep them only at the repo as is.

File: packages/react/README.md (hunk 2 above)
Comment (SokratisVidros) [reply packages/react/README.md:110]: Good catch!

File: packages/react/README.md [hunk 5]
```diff
@@ -1,7 +1,128 @@
...
+- Open a [GitHub support issue](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=question&template=ask_a_question.md&title=Support%3A+)
+- Contact options listed on [our Support page](https://clerk.dev/support?utm_source=github&utm_medium=clerk_react)
+
+## Contributing
+
+We're open to all community contributions! If you'd like to contribute in any way, please read [our contribution guidelines](docs/CONTRIBUTING.md).
+
+## Security
+
+`@clerk/clerk-react` follows good practices of security, but 100% security cannot be assured.
```
Comment (SokratisVidros) [inline packages/react/README.md:114]: 🙃  This phrasing seems a bit scary. What are we trying to say in this section?

File: packages/react/README.md (hunk 1 above)
Comment (devchampian) [reply packages/react/README.md:76]: It has nothing to do with CRA. `render` is the main ReactDOM method to render a React app into the DOM.

File: packages/react/README.md (hunk 3 above)
Comment (devchampian) [reply packages/react/README.md:124]: That's not a missing link. That is a link to the local LICENSE file.

File: packages/react/README.md (hunk 2 above)
Comment (devchampian) [reply packages/react/README.md:110]: That I can do. Good point.

File: packages/react/docs/CONTRIBUTING.md (hunk 4 above)
Comment (devchampian) [reply packages/react/docs/CONTRIBUTING.md:73]: We really want a consistent `CONTRIBUTING.md` file for all the open-source packages so they don't vary by package. Which sections are you saying to remove?

File: packages/react/README.md (hunk 5 above)
Comment (devchampian) [reply packages/react/README.md:114]: It follows the language of the MIT license around the as-is warranty and is based on the Security sections generated by this [GitHub template](https://github.com/dec0dOS/amazing-github-template). It has to do with security liability in the event a bug is introduced and someone using our software wants to sue us for it, but IANAL so open to alternative phrasing.

Comment (SokratisVidros) [reply packages/react/README.md:114]: I see. I don't feel strong, it just seemed weird at first sight.

**Review by SokratisVidros — APPROVED** [review]

**Comment by SokratisVidros** [comment]
@devchampian Rebasing with main should fix the failing test so that you can merge your PR.

**Pull request merged**